src/__tests__/fcnnls.test.ts

```diff
@@ -35,27 +36,25 @@ for (const line of lines) {
 let target = new Matrix(b);
 
 target = target.transpose();
-let solution: Matrix;
-let result: Matrix;
 
-afterEach(() => {
+// used for most tests here and in fcnnlsVector.test.ts
+export function assertResult(result: Matrix, solution: Matrix, precision = 4) {
```

Comment:
I think you need to write this in a separate file (to avoid the tests below to run as a side-effect of importing this file)